cdap-ui/app/cdap/components/FieldLevelLineage/v2/Context/FllContext.tsx

```diff
@@ -17,46 +17,131 @@
 import React from 'react';
 import data from './sample_response';
 import {
-  parseRelations,
-  makeTargetNodes,
+  makeTargetFields,
+  IField,
+  ILink,
+  ITableFields,
+  getFieldsAndLinks,
 } from 'components/FieldLevelLineage/v2/Context/FllContextHelper';
+import * as d3 from 'd3';
 
-export interface INode {
-  id: string;
-  name: string;
-  group: number;
+export const FllContext = React.createContext<IContextState>({} as IContextState);
```

Comment:
nit: May be we should pass in an object with default values for all the required properties in the `IContextState` type. That would help us avoid type aliasing.